### ISSUE TITLE
WSDL importer

### ISSUE THREAD

**Issue by nataliapanferova** [luckymarmot/API-Flow#43]
requested by a user

**Comment by mittsh** [comment]
We have a WADL Importer https://github.com/luckymarmot/Paw-WADLImporter for RESTful APIs, so maybe that's not too far from WSDL – both XML at least! If that help in any way. This WADL importer is actually based on https://github.com/lucybot/api-spec-converter, and quite hacky unfortunately.

**Comment by JonathanMontane** [comment]
The WSDL parser will not be included in API-Flow, as the SOAP format differs too much from the other format in API-Flow

**Comment by JonathanMontane** [comment]
small update:

here is a list of requirements that need to be done before we can implement a WSDL parser:
- need an XMLParser
- SOAP support
- reference model needs an XML Schema model. We can probably inspire us from https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#xmlObject

Once we have this, we can also probably do a WSDL serializer though. (quick reminder that WSDL is actually independent from SOAP, and can describe APIs that use http)

**Comment by dvision1979** [comment]
Any thoughts on this? It's been more than 2 years.

**Comment by lonkelle** [comment]
I agree, I still work with companies that use this old format. It looks like there is still some interest in this feature, at least.
